AssertionError accessing node content using "nodecontent" (no other errors)

From a fresh Julia session:

```
julia> using EzXML, HTTP

julia> text = String(HTTP.get("https://www.postgresql.org/docs/current/errcodes-appendix.html"));

julia> html = parsehtml(text)
EzXML.Document(EzXML.Node(<HTML_DOCUMENT_NODE@0x00007f94156b78d0>))

julia> code = findfirst("//table[@summary='PostgreSQL Error Codes']/tbody/tr/td/code", html)
EzXML.Node(<ELEMENT_NODE@0x00007f94179f39a0>)

julia> nodecontent(code)
ERROR: AssertionError: isempty(XML_GLOBAL_ERROR_STACK)
Stacktrace:
 [1] macro expansion at /Users/ericdavies/.julia/packages/EzXML/r19gO/src/error.jl:49 [inlined]
 [2] nodecontent(::EzXML.Node) at /Users/ericdavies/.julia/packages/EzXML/r19gO/src/node.jl:1120
 [3] top-level scope at none:0
```

No other errors or warnings were generated.

Using EzXML v0.9.1 and Julia 1.1.1

--- a/src/document.jl
+++ b/src/document.jl
@@ -110,6 +110,7 @@ function parsehtml(htmlstring::AbstractString)
         Ptr{_Node},
         (Cstring, Cint, Cstring, Cstring, Cint),
         htmlstring, sizeof(htmlstring), url, encoding, options) != C_NULL
+    show_warnings()
     return Document(doc_ptr)
 end
 

